tee-worker/service/src/sync_state.rs

```diff
@@ -86,10 +86,17 @@ async fn get_enclave_url_of_first_registered<NodeApi: PalletTeerexApi, EnclaveAp
 	enclave_api: &EnclaveApi,
 ) -> Result<String> {
 	let self_mr_enclave = enclave_api.get_mrenclave()?;
+	let all_scheduled_mr_enclaves = node_api.all_schedule_mr_enclaves(None)?;
+	if all_scheduled_mr_enclaves.is_empty() {
+		return Err(Error::NoParachainScheduledMrEnclaves)
+	}
```

Comment:
`contains` should be enough I guess?

Reply:
yes, but if there is no scheduled enclave in the parachain, the error should be different from the `InvalidEnclave` I think.